### DISCUSSION TITLE
知识库搜索功能

### DISCUSSION THREAD

**Discussion by Github-CJX** [labring/FastGPT#627] [Q&A]
本地化部署的FastGPT为什么只有一个语义检索，而官方网站有三种搜索方式，分别是语义检索，增强语义检索以及混合检索，文档中也没有提到这个是不是商业版独有功能

**Comment by c121914yu** [comment]
> 本地化部署的FastGPT为什么只有一个语义检索，而官方网站有三种搜索方式，分别是语义检索，增强语义检索以及混合检索，文档中也没有提到这个是不是商业版独有功能

是的，目前没开源。

**Reply by gaord** [reply]
有计划放出来吗？语义检索出来的排序结果有时候不对呀
